near-sdk-sim/src/runtime.rs

```diff
@@ -347,7 +347,12 @@ impl RuntimeStandalone {
     }
 
     /// Returns a ViewResult containing the value or error and any logs
-    pub fn view_method_call(&self, account_id: &str, method_name: &str, args: &[u8]) -> ViewResult {
+    pub fn view_method_call(
```

Comment:
should we change the name here to `view_function_call`? Or is this PR only for args

Reply:
Ah I meant to mention this in the description. I didn't change anything in sdk-sim because it's being deprecated and no reason to create api breakages for something like this